appveyor.yml

```diff
@@ -34,12 +31,15 @@ install:
   - conda update -q conda
   - conda config --add channels conda-forge
   - conda info -a
-  - "conda create -q -n test-environment python=%PYTHON_VERSION%"
-  - activate test-environment
-  - conda install pandas shapely=1.5 fiona pyproj rtree matplotlib descartes
-  - conda install pytest mock
+  - "conda create --quiet --name test-environment python=%PYTHON_VERSION% pandas msinttypes --file requirements.txt --file requirements.test.txt"
```

Comment:
This is the same changes from master but now with `msinttypes` to add `stdint.h` for Python 2.7 extensions.
We don't need that when compiling with vc 14 (Python >=3.5).